cmd/argocd/commands/cluster.go

```diff
@@ -217,7 +219,7 @@ func NewClusterRemoveCommand(clientOpts *argocdclient.ClientOptions) *cobra.Comm
 
 			for _, clusterName := range args {
 				// TODO(jessesuen): find the right context and remove manager RBAC artifacts
```

Comment:
TODO still relevant?

Reply:
Yes, we currently leave behind the argocd-manager service account, role, rolebinding during a `cluster rm`. We should eventually have an option during `cluster rm` to delete these artifacts.